reps/2023-10-13-accelerator-support.md

```diff
@@ -0,0 +1,234 @@
+# Ray Accelerator Support
+
+## Summary
+
+Holistic design of supporting accelerators in Ray.
+This also serves as the reference doc for contributors
+to add new accelerator support in the future.
+
+### General Motivation
+
+Nowadays, more and more specialized accelerators (e.g. TPU, HPU) are coming out to speed up AI workloads.
+Supporting those accelerators natively in Ray will increase Ray's adoption as the computing framework for scaling AI applications.
```

Comment:
We should define what "supporting them natively" means at a high level. i.e., what does native support enable?

Reply:
I'll add more details. But basically the same support level as existing Nvidia GPU.